src/js/components/chart/utils-normalization.js

```diff
@@ -0,0 +1,219 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import zip from 'lodash/zip';
+import flatten from 'lodash/flatten';
+import isInteger from 'lodash/isInteger';
+import sortedIndexOf from 'lodash/sortedIndexOf';
+
+
+/**
+ * The normalization class includes a normalize method in order to help
+ * the Chart consumers to normalize the series.
+ *
+ * Chart component assumes the array of values
+ * (whether its for Line/Bar/Area/Axis) is always normalize and the points are
+ * evenly spreader on the array.
+ *
+ * normalize func will normalize values in case of data-gaps and blank areas,
+ * and will fill 'undefined' for the missing points added during normalization
+ */
+
+export class normalization {
```

Comment:
I don't think this really needs to be a class. I think we can `export default function normalize()` and have all other functions just be local to this file.